### ISSUE TITLE
Feature Request: newline-condition

### ISSUE THREAD

**Issue by zheeeng** [palantir/tslint#3995]
#### Rule: newline-condition

prefer

```ts
if (condition1
    || codition2
    || condition3
) {}

const pass = condition1
    && condition2
    && condition3
```

over

```ts
if (condition1 || codition2 || condition3) {}

const pass = condition1 && condition2 && condition3
```

### Rationale

When we blame someone a condition statement, we can get the reason why this condition is added through the git commit message.

**Comment by giladgray** [comment]
@zheeeng this should probably be an option on an existing rule, tho i'm not sure the ideal choice off the top of my head. we have several that deal with whitespace and newlines.

Prettier handles this elegantly (if your conditions are longer than a single line). https://github.com/alexjoverm/tslint-config-prettier

**Comment by johnwiseheart** [comment]
Going to close this in favor of `prettier`.
